### PR TITLE
#80 Rename event types

### PR DESCRIPTION
**Pull request by Andersos**
@jarib raised some concerns with the database so we need to figure that out.

The problem will be that some databases will be using the old naming so we need to have a migration.

### CODE REVIEW

**Comment by Andersos** [comment]
I'll have a new look at this next week. 
https://github.com/finn-no/unleash#making-a-schema-change

**Comment by ivarconr** [comment]
just do a db-migrate which renames the event-names. :+1:

File: lib/eventType.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 module.exports = {
     featureCreated : 'feature-created',
     featureUpdated : 'feature-updated',
-    featureArchive : 'feature-archive',
-    featureRevive  : 'feature-revive',
+    featureArchive : 'feature-archived',
+    featureRevive  : 'feature-revived',
```
Comment (jarib) [inline lib/eventType.js]: The keys in this object should use the new name as well.

Comment (Andersos) [inline lib/eventType.js]: :+1:

File: migrations/sql/006-rename-eventtype.down.sql [hunk 2]
```diff
@@ -0,0 +1,2 @@
+UPDATE events SET type=feature-revive WHERE type=feature-revived;
+UPDATE events SET type=feature-archive WHERE type=feature-archived;
```
Comment (jarib) [inline migrations/sql/006-rename-eventtype.down.sql]: The types need to be single-quoted. Have you run the migration locally (`db-migrate up`)?

File: migrations/sql/006-rename-eventtype.up.sql [hunk 3]
```diff
@@ -0,0 +1,2 @@
+UPDATE events SET type=feature-revived WHERE type=feature-revive;
+UPDATE events SET type=feature-archived WHERE type=feature-archive;
```
Comment (jarib) [inline migrations/sql/006-rename-eventtype.up.sql]: Ditto

File: migrations/sql/006-rename-eventtype.down.sql (hunk 2 above)
Comment (Andersos) [inline migrations/sql/006-rename-eventtype.down.sql]: I have not run db-migrate up yet. I didn't have postgres locally but I will install http://postgresapp.com/ as you recommended =)

**Comment by jarib** [comment]
LGTM when the SQL is fixed.

**Pull request merged**